src/Microsoft.Health.Fhir.Stu3.Core/Features/Operations/Export/ExportAnonymizerFactory.cs

```diff
@@ -0,0 +1,63 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Anonymizer.Core;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Export
+{
+    public class ExportAnonymizerFactory : IAnonymizerFactory
```

Comment:
It doesn't seem ideal that we have all these files in two locations

Reply:
If the goal is to not support R5, would it be simpler to detect the FHIR version in the AnonymizationModule and not add the services if it is R5?

Reply:
Check the version in code not work for this scénario, it would failed at compile code. We do not have R5 Nuget now. 

I updated the PR with below change to work around this issue before R5 anonymizer release:

1. Remove code file from shared.*..projitems
2. Add None reference in shared.*.shproj
3. Add file link reference in R4 and STU3 project.

Not sure any concern for this solution? Thanks!